pkg/label/label.go

```diff
@@ -0,0 +1,15 @@
+// Package label contains common Kubernetes object labels. These are defined in
+// https://github.com/giantswarm/fmt/blob/master/kubernetes/annotations_and_labels.md.
+package label
```

Comment:
Adding a label package like we have in cluster-operator. As app-operator has 2 controllers. So we need shared code.